package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "lint": "standard",
-    "test": "mocha"
+    "test": "mocha",
+    "prepublishOnly": "mkdir docs; barnard59-docs local > docs/readme.md",
```

Comment:
Also update docs file when packaging in case anyone would edit the manifest on GitHub directly (not sure if that runs hooks)